lib/npm-installation-manager.ts

```diff
@@ -70,6 +70,14 @@ export class NpmInstallationManager implements INpmInstallationManager {
 
 	private installCore(packageName: string, pathToSave: string, version: string, dependencyType: string): IFuture<string> {
 		return (() => {
+			if(packageName.indexOf(".") === 0 || packageName.indexOf(".tgz") >= 0) {
+				let cwd = process.cwd();
+				let possiblePackageName = path.join(`${cwd}`, packageName);
```

Comment:
instead of this you can use `path.resolve(packageName)` - this will work with both full and relative paths. In this case you can skip the above if as well and use:
```TypeScript
const possiblePackageName= path.resolve(packageName);
if (this.$fs.exists(possiblePackageName).wait()) {
...
}